proxy/envoy/policy.go

```diff
@@ -61,6 +61,7 @@ func insertMixerFilter(listeners []*Listener, instances []*model.ServiceInstance
 							"source.ip":  context.IPAddress,
 							"source.uid": context.UID,
 						},
+						QuotaName: "RequestCount",
```

Comment:
Here's my alpha proposal:

- hardcode in a name here (maybe change it from `RequestCount` to `RateLimit`).
- add a field on the mixer quota aspect config called `limit` which allows specification of a limit that should be applied within that scope/subject
- remove `max_amount` from the descriptor of `RequestCount`/`RateLimit` (no sense in setting an upper bound on rate limits at this point)
- update the quota manager code to use `limit` instead of `max_amount`
- set `limit: 5` in the aspect config for the selector of "ratings*"

That should allow mixerclient to call `Quota()` on all requests, but have mixer just return `status.OK` unless the quota aspect is configured and linked to the `RateLimit` descriptor for a scope/subject.  And it should allow *different* limits based on scope/subject.

Thoughts?